master/reference/calicoctl/resources/policy.md

```diff
@@ -66,14 +66,17 @@ connection tracking is always used for flows to and from those.
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| action      | Action to perform when matching this rule. | allow, deny, log | string | |
+| action      | Action to perform when matching this rule. | allow, deny, log, pass | string | |
 | protocol    | Positive protocol match.  | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | notProtocol | Negative protocol match. | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | icmp        | ICMP match criteria.     | | [ICMP](#icmp) | |
 | notICMP     | Negative match on ICMP. | | [ICMP](#icmp) | |
 | source      | Source match parameters. |  | [EntityRule](#entityrule) | |
 | destination | Destination match parameters. |  | [EntityRule](#entityrule) | |
 
+An `action` of `pass` will cause the connection to be processed by the policy
```

Comment:
Don't think we should be talking about connections here.  How about something like:

An `action` of `pass` will skip over the remaining Policy Resource matches and jump to the Profile resource assigned to the endpoint, applying the policy rules configured in the Profile.  If there is no Profile configured for the endpoint the default applied action is `deny`.